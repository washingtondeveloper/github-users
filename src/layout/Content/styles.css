.gs-content {
  background: #f1f1f1;
  margin: 0 10px;
  box-shadow: 0 0 3px rgba(0, 0, 0, 0.5);
  border-radius: 5px;
}
