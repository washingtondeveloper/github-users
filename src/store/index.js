import { createStore, applyMiddleware } from "redux";
import reduxThunk from "redux-thunk";

import rootReducer from "./ducks";

export default createStore(rootReducer, applyMiddleware(reduxThunk));
