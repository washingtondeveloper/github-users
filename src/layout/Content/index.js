import React from "react";

import "./styles.css";

export default ({ children }) => <div className="gs-content ">{children}</div>;
