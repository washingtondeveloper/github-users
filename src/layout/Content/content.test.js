import React from "react";

import renderer from "react-test-renderer";
import { Provider } from "react-redux";
import store from "@/store";

import Content from "./";

describe("Component Content", () => {
  test("Deve renderizar sem erros", () => {
    const tree = renderer
      .create(
        <Provider store={store}>
          <Content />
        </Provider>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
