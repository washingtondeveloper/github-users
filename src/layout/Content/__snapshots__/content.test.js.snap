// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component Content Deve renderizar sem erros 1`] = `
<div
  className="gs-content "
/>
`;
